src/foam/u2/view/ScrollTableView.js

```diff
@@ -232,12 +301,14 @@
         disableUserSelection: this.disableUserSelection,
         multiSelectEnabled: this.multiSelectEnabled,
         selectedObjects$: this.selectedObjects$
-      },  this, this.__subSubContext__.createSubContext({ memento: this.currentMemento_ ? this.currentMemento_.tail : this.currentMemento_ }));
-      
+      }, this, this.__subSubContext__.createSubContext({ memento: this.currentMemento_ ? this.currentMemento_.tail : this.currentMemento_ }));
```

Comment:
Why can't we just export: the currentMemento_ tail? 

This line would be shorter as this.currentMento_ && this.currentMento_.tail

Reply:
It's not necessary that we have a memento for all scroll table views (inline tables for now don't have mementos) but will replace with this.currentMemento_ && this.currentMemento_.tail